zipline/data/benchmarks.py

```diff
@@ -45,17 +42,11 @@ def get_benchmark_returns(symbol, first_date, last_date):
     """
     data = pd_reader.DataReader(
         symbol,
-        'google',
+        'yahoo',
```

Comment:
So the newer version works ok with yahoo again?

Reply:
Yes on my machine with a fresh zipline install it works.